src/main/java/org/example/XMLValidator.java

```diff
@@ -0,0 +1,162 @@
+package org.example;
+
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+import javax.xml.XMLConstants;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.Source;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+import java.io.File;
+import java.io.IOException;
+
+
+public class XMLValidator {
+    private static final Logger logger = LogManager.getLogger(XMLValidator.class);
+    public static void main(String[] args) throws SAXException, IOException, ParserConfigurationException {
```

Comment:
I think it's not very correct to throw an exception in the main method signature. Instead, I think it'd better to use try-catch. It was one of Vladislav's requirements regarding my previous assignment. The rest is good!